src/main/java/org/jabref/gui/preferences/PreferencesSearchHandler.java

```diff
@@ -0,0 +1,111 @@
+package org.jabref.gui.preferences;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javafx.beans.property.ListProperty;
+import javafx.beans.property.SimpleListProperty;
+import javafx.beans.property.StringProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.scene.Node;
+import javafx.scene.Parent;
+import javafx.scene.control.Labeled;
+
+class PreferencesSearchHandler {
+
+    private final ObservableList<PrefsTab> preferenceTabs;
+    private final StringProperty searchText;
+    private final ListProperty<PrefsTab> filteredPreferenceTabs;
+    private final Map<String, Set<PrefsTab>> preferenceTabsLabelNames;
+
+    PreferencesSearchHandler(ObservableList<PrefsTab> preferenceTabs, StringProperty searchText) {
```

Comment:
it should be sufficient to have the parameter of Interface List

Reply:
Youre right 👍